apps/studio/src/referralHeroConfig.ts

```diff
@@ -0,0 +1,17 @@
+const loadReferralHeroScript = () => {
+  const script = document.createElement("script");
+
+  script.textContent = `
+  !function(m,a,i,t,r,e){if(m.RH)return;r=m.RH={},r.uuid
+  =t,r.loaded=0,r.base_url=i,r.queue=[],m.rht=function()
+  {r.queue.push(arguments)};e=a.getElementsByTagName('script')
+  [0],c=a.createElement('script');c.async=!0,c.src=
+  'https://referralhero-global-code.s3.amazonaws.com/'+'production'+
+  '/'+t+'.js',e.parentNode.insertBefore(c,e)}(window,document,
+  'https://app.referralhero.com/','RHa857ddbc7a');
```

Comment:
Should this UUID be organized in a .env file or Constant? It isn't a secret key, just a tracker ID, but there might be multiple campaigns. Could be a problem for future integration and refactoring but right now to keep it simple I left it in the script for this initial campaign.